app/controllers/admin/events/list.js

```diff
@@ -65,6 +65,18 @@ export default Controller.extend({
     editEvent(id) {
       this.transitionToRoute('events.view.edit.basic-details', id);
     },
+    restoreEvent(id) {
+      this.store.query('event', {
+        filter: [{
+          name : 'identifier',
+          op   : 'eq',
+          val  : id
+        }],
+        get_trashed: true }).then(function(event) {
```

Comment:
have you tried passing `get_trashed` as a filter?
like,
```js
{ 
name : 'get_trashed',
op: 'eq',
val: true
}
```

Reply:
@CosmicCoder96 I tried this but it is giving an error.
![image](https://user-images.githubusercontent.com/22127980/44157039-11140800-a0cf-11e8-97e6-c8e86fc77075.png)

Reply:
https://user-images.githubusercontent.com/22127980/44157039-11140800-a0cf-11e8-97e6-c8e86fc77075.png